load/FilenameParser.py

```diff
@@ -488,12 +488,6 @@ def __init__(self, config):
 	def process3(self, filesets):
 		db = SQLUtility(self.config)
 		self.chapterMap = db.selectMap("SELECT id, chapters FROM books", None)
-		## I am not certain the LXX actually exists
-		extras = {"FRT":6, "INT":1, "BAK":2, "LXX":1, "CNC":2, "GLO":26, "TDX":1, "NDX":1, "OTH":5, 
-			"XXA":4, "XXB":3, "XXC":1, "XXD":1, "XXE":1, "XXF":1, "XXG":1}
-		self.chapterMap.update(extras)
-		corrections = {"MAL":3, "MAN":1, "PS2":1, "BAR":5, "EZA":9, "SIR":52}
-		self.chapterMap.update(corrections)
```

Comment:
let's also remove maxChapterMap update (for JOL, PSA, MAL and BAR) and usfx2Map update

Reply:
@bradflood I have removed the changes for `JOL, PSA, MAL and BAR` and the changes for the usfx2Map list